edge-hub/docker/linux/amd64/Dockerfile

```diff
@@ -22,14 +26,19 @@ COPY --from=builder publish/* /usr/local/lib/
 
 WORKDIR /app	
 
-COPY $EXE_DIR/ ./	
+COPY $EXE_DIR/ ./
+
+# Set broker log level
+ENV RUST_LOG=debug
 
 # Expose MQTT, AMQP and HTTPS ports	
+EXPOSE 1883/tcp
 EXPOSE 8883/tcp	
 EXPOSE 5671/tcp	
 EXPOSE 443/tcp	
 
 USER edgehubuser	
 
-CMD echo "$(date --utc +"%Y-%m-%d %H:%M:%S %:z") Starting Edge Hub" && \	
-    exec /usr/bin/dotnet Microsoft.Azure.Devices.Edge.Hub.Service.dll
+CMD echo "$(date --utc +"%Y-%m-%d %H:%M:%S %:z") Starting Edge Hub" && \
+    cd /tmp/mqtt && \
+    /usr/local/bin/watchdog -c /tmp/mqtt/config/config.json
```

Comment:
As discussed offline, we probably should not pass any mqttd related params to watchdog, instead have mqttd look for default config location.